.github/workflows/osx.yml

```diff
@@ -16,6 +16,7 @@ jobs:
         ./contrib/utilities/download_clang_format
         # force cmake version:
         pip install cmake==3.18.4
+        echo "/Library/Frameworks/Python.framework/Versions/2.7/bin" >> $GITHUB_PATH
```

Comment:
I think it would be good to add a comment to these two lines to explain *why* we use this specific cmake version and why you need to write something into this file. (Incidentally, what do you actually do here? A `PATH` suggests that the environment variable contains the name of a directory, so what does writing into/append to it doing?)

Reply:
This was already changed in https://github.com/dealii/dealii/pull/11384 again. :wink: